modules/machineset-non-guaranteed-instance.adoc

```diff
@@ -1,22 +1,53 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating_machinesets/creating-machineset-aws.adoc
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
 
-[id="machineset-non-guaranteed-instance_{context}"]
-= MachineSet Spot Instances
+ifeval::["{context}" == "creating-machineset-aws"]
+:aws:
+endif::[]
+ifeval::["{context}" == "creating-machineset-azure"]
+:azure:
+endif::[]
+ifeval::["{context}" == "creating-machineset-gcp"]
+:gcp:
+endif::[]
 
+[id="machineset-non-guaranteed-instance_{context}"]
+ifdef::aws[= MachineSet Spot Instances]
+ifdef::azure[= MachineSet Spot VMs]
+ifdef::gcp[= MachineSet preemptible VM instances]
+ifdef::aws[]
 You can save on costs by creating an AWS MachineSet that deploys machines as non-guaranteed Spot Instances.
```

Comment:
> I would say 'a MachineSet running on AWS', because my view is that this is an OpenShift MachineSet running on AWS. But, I could be wrong because I'm not too familiar with the MachineSet terminology.

@joelspeed, is it accurate to change the wording from "an AWS MachineSet" to "a MachineSet running on AWS"? Thanks!